app/src/main/res/values-fr/strings.xml

```diff
@@ -95,7 +95,7 @@
   <string name="title_activity_contributions">Mes téléversements récents</string>
   <string name="contribution_state_queued">Mis en file d’attente</string>
   <string name="contribution_state_failed">Échec</string>
-  <string name="contribution_state_in_progress">%1$d %% effectué</string>
+  <string name="contribution_state_in_progress">%1$d %% effectué</string>
```

Comment:
Would you mind not including these strings.xml changes in the pull request? Just overwrite using the original files from the starting point of your branch.

Reply:
@nicolas-raoul I tried to revert these changes but it did not seem to be the correct way! Is it possible for you to ignore these files when you squash and merge?

Reply:
Please `git clone` master to a temporary folder, then copy the strings.xml from that master folder, to overwrite the ones in your repository. Then commit and these differences will disappear.

Reply:
Should be fixed now!